MapleServer2/Types/Fishing.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Linq;
+using Maple2Storage.Types.Metadata;
+using MapleServer2.Data.Static;
+using MapleServer2.Enums;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+
+namespace MapleServer2.Types
+{
+    public class Fishing
+    {
+        public int FishId;
+        public int TotalCaught;
+        public int TotalPrizeFish;
+        public int LargestFish;
+        public Fishing() { }
```

Comment:
Don't need to add the default constructor, `AddExistingFish()` and `AddNewFish()` can be static